debian/DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Source: snowflake
+Section: devel
+Priority: optional
+Homepage: https://github.com/keroserene/snowflake
```

Comment:
Better to use https://trac.torproject.org/projects/tor/wiki/doc/Snowflake or https://gitweb.torproject.org/pluggable-transports/snowflake.git.

Reply:
thanks, changed